src/applications/letters/containers/AddressSection.jsx

```diff
@@ -361,6 +367,9 @@ export class AddressSection extends React.Component {
       addressContent = (
         <div className="step-content">{addressUpdateUnavailable}</div>
       );
+      // override needed for view letters button so that users can
+      // still view letters if they end up in this terminal state
+      forceEnableViewLettersButton = true;
```

Comment:
would this ever get set to false? just wondering if this is the clearest say to designate the weird UX

Reply:
nvm I realize that the initialization is falsy, maybe just explicitly set it to false above (line 360)

Reply:
yep I'll do that for clarity